sklearn/mixture/gmm.py

```diff
@@ -112,7 +114,7 @@ def sample_gaussian(mean, covar, covariance_type='diag', n_samples=1,
     return (rand.T + mean).T
 
 
-class GMM(BaseEstimator):
+class _GMMBase(_MixtureBase):
```

Comment:
In that case, I have to send all the code of `_MixtureBase` and `GaussianMixture` in the same PR. 
To simplify the review, I can propose to split the code in different commit. I don't see a better way of not pushing 1000 lines at once.